src/test/ui/issue-13483.rs

```diff
@@ -14,9 +14,3 @@ fn main() {
     } else {
     };
 }
-
-fn foo() {
-    if true {
-    } else if { //ERROR: MISSING CONDITIONAL
```

Comment:
Hmm, so I guess my change means we no longer catch this case? That does seem unfortunate -- I hadn't quite thought that far ahead. :) We could handle this by expanding the set of things we check for, I guess -- e.g., including `;` and `}` and a few other such tokens. Not sure if that's worth it.